scss/main.scss

```diff
@@ -102,3 +102,5 @@ nav {
     }
   }
 }
+
+@import 'responsive';
```

Comment:
@yegor256 `@media` queries don't get to override the styles, like using a class, id or !important does. POC: http://jsfiddle.net/1s8kn0rh/